### ISSUE TITLE
Untranslatable strings on login screen

### ISSUE THREAD

**Issue by phriedrich** [RocketChat/Rocket.Chat.ReactNative#2620]
### Description:

On the screen to join an existing workspace, two strings are hardcoded and can't be translated. Please see attached screenshot.

They are in the file `app/views/NewServerView/ServerInput/index.js`.

![Screenshot](https://user-images.githubusercontent.com/470153/98524606-b51d9700-2277-11eb-9e18-84e29a663585.png)

### Environment Information:

- Rocket.Chat Server Version: 3.7.1
- Rocket.Chat App Version: 4.12.0.100
- Device Name: iOS Simulator, iPhone, iPad
- OS Version: 14.1

**Comment by sumukhah** [comment]
I am working on the issue.
One thing I have noticed is that 
<b>There is no option to change the language before authentication</b>.
 A user has to authenticate first, then she can change the language through the settings option.
@phriedrich @diegolmello 
I have included the strings to the I18n translations and added them to locales/en.json. Do i need to add translations of the strings manually to other languages (such as rus.json, nl.json e.t.c)? Are there any easy ways?

<img width="314" alt="Screenshot 2020-12-04 at 2 26 57 PM" src="https://user-images.githubusercontent.com/23723464/101144087-d175df00-363d-11eb-838f-b1966b3e1e2f.png">
